docs/inquiries/overview.md

```diff
@@ -0,0 +1,6 @@
+Overview
+--------
+
+There are multiple ways you can work with us depending on your needs we can
```

Comment:
_punctuation-for-readability_: There are multiple ways you can work with us **:** depending on your needs **,** we can ...

Reply:
@mikebdp2 for this kind of fixes, please use [Github suggestions](https://haacked.com/archive/2019/06/03/suggested-changes/)


```suggestion
There are multiple ways you can work with us: depending on your needs, we can
```